test/unit/DB.spec.ts

```diff
@@ -20,14 +19,16 @@ const loggers = Logger.createLoggers(Level.Warn);
 const price = 0.005;
 const quantity = 0.1;
 const peerPubKey = '03029c6a4d80c91da9e40529ec41c93b17cc9d7956b59c7d8334b0318d4a86aef8';
+const rHash = '62c8bbef4587cff4286246e63044dc3e454b5693fb5ebd0171b7e58644bfafe2';
+const orderId = uuidv1();
 
 const order: OwnOrder = {
   price,
   quantity,
   isBuy: true,
   createdAt: ms(),
   initialQuantity: quantity,
-  id: uuidv1(),
+  id: orderId,
```

Comment:
Yeah it's true, right now we have a smattering of various methods for generating orders across the suites, it might be nice to have a single one that can be imported where its needed.